In [1]:
# Importing libraries
import pandas as pd

In [2]:
# Importing dataset
df = pd.read_csv('Iris.csv')

In [3]:
# Looking for columns and rows
df.shape

(150, 6)

This shows that this dataset has 150 records(rows) and 6 features(columns).

In [5]:
# Column names.
df.columns

Index(['Id', 'SepalLengthCm', 'SepalWidthCm', 'PetalLengthCm', 'PetalWidthCm',
       'Species'],
      dtype='object')

In [6]:
# To get the mean of data.
df.describe()

Id  SepalLengthCm  SepalWidthCm  PetalLengthCm  PetalWidthCm
count  150.000000     150.000000    150.000000     150.000000    150.000000
mean    75.500000       5.843333      3.054000       3.758667      1.198667
std     43.445368       0.828066      0.433594       1.764420      0.763161
min      1.000000       4.300000      2.000000       1.000000      0.100000
25%     38.250000       5.100000      2.800000       1.600000      0.300000
50%     75.500000       5.800000      3.000000       4.350000      1.300000
75%    112.750000       6.400000      3.300000       5.100000      1.800000
max    150.000000       7.900000      4.400000       6.900000      2.500000

According to above data mean of different columns is:

Id: 75.5

SepalLengthCm: 5.8

SepalWidthCm: 3.05

PetalLengthCm: 3.7

PetalWidthCm: 1.1